mail_composer_cc_bcc/wizards/mail_compose_message.py

```diff
@@ -0,0 +1,183 @@
+                rendered_values = composer._generate_template_for_composer(
+                    res_ids,
+                    # DIFFERENT FROM ODOO NATIVE:
+                    {"email_to", "partner_ids"},
+                    find_or_create_partners=True,
+                )[res_ids[0]]
+                if rendered_values.get("partner_ids"):
+                    composer.partner_ids = rendered_values["partner_ids"]
+            elif composer.parent_id and composer.composition_mode == "comment":
+                composer.partner_ids = composer.parent_id.partner_ids
+            elif not composer.template_id:
+                composer.partner_ids = False
+
+    # ------------------------------------------------------------
+    # RENDERING / VALUES GENERATION
+    # ------------------------------------------------------------
+
+    def _prepare_mail_values_rendered(self, res_ids):
+        """
+        add cc and bcc when send to mail.message
+        """
+        mail_values = super()._prepare_mail_values_rendered(res_ids)
+
+        for res_id in mail_values:
+            mail_values[res_id].update(
+                {
+                    "recipient_cc_ids": self.partner_cc_ids.ids,
+                    "recipient_bcc_ids": self.partner_bcc_ids.ids,
+                }
+            )
+        return mail_values
+
+    # ------------------------------------------------------------
+    # ACTIONS
+    # ------------------------------------------------------------
+
+    def _action_send_mail(self, auto_commit=False):
+        """
+        Add context is_from_composer when composition_mode == 'comment'
+        """
+        result_mails_su, result_messages = (
+            self.env["mail.mail"].sudo(),
+            self.env["mail.message"],
+        )
+
+        for wizard in self:
+            if wizard.res_domain:
+                search_domain = wizard._evaluate_res_domain()
+                search_user = wizard.res_domain_user_id or self.env.user
+                res_ids = (
+                    self.env[wizard.model]
+                    .with_user(search_user)
+                    .search(search_domain)
+                    .ids
+                )
+            else:
+                res_ids = wizard._evaluate_res_ids()
+            # in comment mode: raise here as anyway message_post will raise.
+            if not res_ids and wizard.composition_mode == "comment":
+                raise ValueError(
+                    _(
+                        "Mail composer in comment mode should run on at least one record. No records found (model %(model_name)s).",
+                        model_name=wizard.model,
+                    )
+                )
+
+            if wizard.composition_mode == "mass_mail":
+                result_mails_su += wizard._action_send_mail_mass_mail(
+                    res_ids, auto_commit=auto_commit
+                )
+            else:
+                # DIFFERENT FROM ODOO NATIVE:
+                context = {
```

Comment:
This change is imperative. Instead of augmenting the context, it's being overwritten. This leads to unintended consequences in the sales module, such as orders not being flagged as sent upon sending the quote, due to the replacement of the context key that the sales module relies on.

Reply:
hello @llabusch93, I updated as you suggest :pray: